### ISSUE TITLE
predict function for test set new observations

### ISSUE THREAD

**Issue by speiser10** [bcjaeger/bimm#10]
bimm_predict() with a test dataset of new_obs (new observations) outputs odds ratios, but it should output predicted probabilities to be consistent with the bimm_predict() for test data of new subjects. can you please edit this?

**Comment by speiser10** [comment]
found another issue with the predict function for new observations. it only works if you predict on the entire training dataset. I just tested it out for doing a further split of the training data where one obs per person was held out and then I tried using the predict function with "new_obs" and it's giving an error Error in data.frame(..., check.names = FALSE) :   arguments imply differing number of rows: 5386, 250

I think it might be resolved by including allow.new.levels=TRUE in the predict function for that, but I'm not totally sure.

**Comment by speiser10** [comment]
made some edits to the predict function that should address this. will test it out tomorrow and close this if it is completely resolved.

**Comment by speiser10** [comment]
Byron--I can confirm the predict function works now (had to update today a bit). Can you please merge my changes into the main bimm repository? Thanks!

**Comment by speiser10** [comment]
fixed with edits to the predict function for new obs

**Comment by bcjaeger** [comment]
🥳 @speiser10 That's awesome! I don't see an open pull request (PR) for this. Can I help you open a PR?

**Comment by speiser10** [comment]
my bad. Just did a pull request, I think?
